scss/bitstyles/layout/relative/relative.stories.mdx

```diff
@@ -0,0 +1,15 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
+
+<Meta title="Layout/Relative" />
+
+# Relative
+
+Gives the element `position: relative`.
```

Comment:
Makes perfect sense, but how does one properly use / view the example? It looks pretty much empty. Maybe adding color and more items makes it easier?

![Uploading Screenshot 2021-02-18 at 10.23.10.png…]()

Reply:
Hmm, yes 🤔 I’ll try to come up with a good example of this